modules/core/src/main/java/org/apache/ignite/internal/processors/cache/CacheAffinitySharedManager.java

```diff
@@ -2920,6 +2921,56 @@ private CacheGroupNoAffOrFilteredHolder createHolder(
         return new CacheGroupNoAffOrFilteredHolder(ccfg.getRebalanceMode() != NONE, cctx, aff, initAff);
     }
 
+    /**
+     * Prints {@code waitInfo} to provided {@code log}.
+     *
+     * @param log Logger to print to.
+     */
+    public void printWaitInfo(IgniteLogger log) {
+        if (waitInfo == null || waitInfo.empty())
+            return;
+
+        try {
+            final String header = "Current affinity assignment is not ideal, it is waiting for cache: ";
```

Comment:
In accordance with https://cwiki.apache.org/confluence/display/IGNITE/Abbreviation+Rules please rename `header` to `hdr`.